L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -0,0 +1,667 @@
+#ifndef L1Trigger_TrackFindingTracklet_interface_Settings_h
+#define L1Trigger_TrackFindingTracklet_interface_Settings_h
+
+#include <iostream>
+#include <string>
+#include <array>
+#include <set>
+#include <map>
+#include <cassert>
+#include <cmath>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+namespace Trklet {
+
+  class Settings {
+  public:
+    Settings(){
+//Uncomment to run the hybrid algorithm
+#ifdef CMSSW_GIT_HASH
+#define USEHYBRID
+#endif
+    }
+
+    // processing & memory modules, wiring, etc.
+    std::string DTCLinkFile() const {
```

Comment:
here and below, should be returned by `std::string const &`

Reply:
done

Reply:
implemented